src/pages/signin/SignInPageLayout/SignInPageLayoutNarrow.js

```diff
@@ -1,18 +1,32 @@
 import React from 'react';
 import {
+    Image,
     ScrollView, Text, View,
 } from 'react-native';
 import PropTypes from 'prop-types';
 import styles from '../../../styles/styles';
 import variables from '../../../styles/variables';
 import ExpensifyCashLogo from '../../../../assets/images/expensify-cash.svg';
+import welcomeScreenshot from '../../../../assets/images/welcome-screenshot.png';
+import TermsAndLicenses from '../TermsAndLicenses';
+import WelcomeText from '../../../components/WelcomeText';
 
 const propTypes = {
+
     // The children to show inside the layout
     children: PropTypes.node.isRequired,
+
+    // Whether we should show the welcome elements
+    shouldShowWelcomeText: PropTypes.Boolean,
+    shouldShowWelcomeScreenshot: PropTypes.Boolean,
```

Comment:
These are not valid `PropTypes` so we are passing `undefined` here.

https://github.com/facebook/prop-types

Reply:
ahh that totally makes sense, I was confused as to why I was getting warnings.